lib/okapi/cli.rb

```diff
@@ -68,42 +68,51 @@ def execute
     end
 
     subcommand "get", "issue a GET request" do
-      parameter "PATH", "PATH to fetch from the api"
+      parameter "PATH", "PATH of the resource to get"
+      option "--no-tenant", :flag, "Perform the request without a tenant or user token"
+      option "--no-user", :flag, "Perform the request without the user token"
```

Comment:
Couldn't decide what name to give these flags.

`--without-tenant`
`--without-user`

`--no-token`
`--without-token`

...etc